### DISCUSSION TITLE
✅ `supervision-0.13.0` checklist

### DISCUSSION THREAD

**Discussion by SkalskiP** [roboflow/supervision#230] [Contribution]
## overview

Hi everyone 👋🏻! Time to plan `supervision-0.13.0` release. It looks like (once again) we will have many external contributors. 🔥 I have written down the tasks we plan to carry out and those who initially signaled that they want to contribute. Don't take this as binding. I'm just trying to plan the scope of work.

## note

- All issues and pull requests related to `supervision-0.13.0` release got marked with `version: 0.13.0` label and are pinned to `version: 0.13.0` [milestone](https://github.com/roboflow/supervision/milestone/2). 
- Please be patient. 🙏🏻 I love to work on `supervision`, but I have many other responsibilities. I always try to be as responsive as possible.

## checklist

- annotators redesign by @hardikdava and @SkalskiP
   - https://github.com/roboflow/supervision/pull/170
   - https://github.com/roboflow/supervision/issues/143
   - https://github.com/roboflow/supervision/issues/168
   - https://github.com/roboflow/supervision/issues/49
- tracking support by @dbroboflow
   - https://github.com/roboflow/supervision/issues/195
   - https://github.com/roboflow/supervision/pull/203
- fix `sv.DetectionDataset.as_coco` polygon output by @mayankagarwals
   - https://github.com/roboflow/supervision/pull/225
   - https://github.com/roboflow/supervision/issues/205
- PaddlePaddle integration by @hardikdava 
   - https://github.com/roboflow/supervision/pull/191
   - https://github.com/roboflow/supervision/issues/187
- Ultralytics integration by @hardikdava
   - https://github.com/roboflow/supervision/pull/222
   - https://github.com/roboflow/supervision/issues/221
- Show Progress in time-consuming tasks by @mayankagarwals
   - https://github.com/roboflow/supervision/issues/183
- Add support for loading segmentation datasets in Pascal VOC format by @kirilllzaitsev 
- Building test package versions from the `develop` branch using GitHub actions by @onuralpszr 
- Use pre-commit to automate code formatting by @onuralpszr 
- object detection mAP calculation by @hardikdava
   - https://github.com/roboflow/supervision/issues/232

Please let me know in the comment if I forgot or mixed up anything. 🙏🏻

**Comment by onuralpszr** [comment]
Automatic Github action stable release trigger using git tags ( tag example v0.13.0 since we already have tags let's keep them in use as well),  for stable releases. Since we going to make "develop" release let's also make that for stable too. That means no more manual builds

**Reply by SkalskiP** [reply]
@onuralpszr, what would be the triggering mechanism? I do the commands below, and it triggers package build?

```
git tag 0.13.0-alpha
git push origin --tags
```

**Reply by onuralpszr** [reply]
Yes, change your version in toml and add tag then push and action will start.

**Reply by SkalskiP** [reply]
Would that be possible to get away with only tagging? That would be awesome to drop the extra step.
